### ISSUE TITLE
Copy button isn't tabbable

### ISSUE THREAD

**Issue by mAAdhaTTah** [mAAdhaTTah/WP-Gistpen#323]
In the content, the copy button gets skipped. Looks like this is because it's an anchor tag with no href. Should be a button.

**Comment by mAAdhaTTah** [comment]
Fixed by https://github.com/PrismJS/prism/pull/1926
